library/psa_crypto_storage.h

```diff
@@ -231,8 +231,9 @@ typedef uint16_t psa_crypto_transaction_type_t;
  * This type is designed to be serialized by writing the memory representation
  * and reading it back on the same device.
  *
- * \note The transaction mechanism is designed for a single active transaction
- *       at a time. The transaction object is #psa_crypto_transaction.
+ * \note The transaction mechanism is not thread-safe. There can only be one
```

Comment:
This is an internal comment, but if the goal is to make it official (which I agree it with) then we need to also mention that in public documentation. That would be a `\warning` in `mbedtls_config.h` in the documentation of `MBEDTLS_PSA_CRYPTO_SE_C`.